apps/meteor/client/views/admin/apps/AppMenu.js

```diff
@@ -1,11 +1,35 @@
-import { Box, Icon, Menu } from '@rocket.chat/fuselage';
+import { Box, Icon, Menu, Throbber } from '@rocket.chat/fuselage';
 import { useSetModal, useMethod, useEndpoint, useTranslation, useRoute, useRouteParameter } from '@rocket.chat/ui-contexts';
-import React, { useMemo, useCallback } from 'react';
+import React, { useMemo, useCallback, useState } from 'react';
 
+import { Apps } from '../../../../app/apps/client/orchestrator';
 import WarningModal from '../../../components/WarningModal';
+import AppPermissionsReviewModal from './AppPermissionsReviewModal';
 import CloudLoginModal from './CloudLoginModal';
 import IframeModal from './IframeModal';
-import { appEnabledStatuses, warnStatusChange, handleAPIError } from './helpers';
+import { appEnabledStatuses, warnStatusChange, handleAPIError, appButtonProps, handleInstallError } from './helpers';
+
+const installApp = async ({ id, name, version, permissionsGranted }) => {
```

Comment:
Both `installApp` and `actions` are duplicated in AppStatus and AppMenu, doing the same. Isn't it worth exporting them or moving them to another file?

Reply:
Solved